advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -199,7 +199,13 @@ private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulne
     }
 
     val references = references.mapNotNull { reference ->
-        reference.url.trim().toUri().onFailure { e ->
+        reference.url.trim().let { url ->
```

Comment:
Alternatively:

    val url = reference.url.trim()
    url.takeUnless { it.startsWith("://") } ?: "https$url"